tree_sitter_v/grammar.js

```diff
@@ -1111,7 +1097,7 @@ module.exports = grammar({
 				field('right', $.expression_list),
 			),
 
-		block: ($) => seq('{', optional($._statement_list), '}'),
+		block: ($) => seq('{', repeat($._statement), '}'),
```

Comment:
Will that handle `{ f();g();h(); }` and `{}` ?

Reply:
Actually, it is.  `;` support was added to V not too long ago.

`v fmt` will remove the `;`, but it is allowed to be there.